frontend/src/app/pages/exercises/exercise/shared/simulation/signaller-modal/details-modal/signaller-modal-start-transfer-of-category-modal/signaller-modal-start-transfer-of-category-modal.component.ts

```diff
@@ -86,16 +92,34 @@ export class SignallerModalStartTransferOfCategoryModalComponent
             return;
         }
 
-        // This counter must be increased prior to proposing the first action
-        // Otherwise, it might become 0 in between, if the first action is handled faster than the second one is proposed
+        // loadingCount has to be fully increased before actually proposing the first action. Otherwise, it could become 0 in between.
+        // Hence, all actions to be proposed are collected here and proposed in the end.
+        // const actionProposers: (() => void)[] = [];
```

Comment:
Remove this line, it seems to be a leftover from before importing `ExerciseAction`.

Reply:
Oh, yes, this shouldn't be there. Removed in 08f28abc69937c959a439b681db7e92012e73488